tola_management/programadmin.py

```diff
@@ -121,7 +121,7 @@ def _result_disaggregation_serializer(raw_disaggs):
         Cell(ws, value=_('Rationale'))
     ]
 
-    title = Cell(ws, value=_("Change log"))
+    title = Cell(ws, value=_("Program change log"))
```

Comment:
It's probably a good idea to add a translator comment, even though this same string is used in another file that has a translator comment. If you want, just copy the same comment and add it here.